styles/bsky/catppuccin.user.less

```diff
@@ -623,6 +623,89 @@
       background-color: @surface0 !important;
       border-color: @overlay0 !important;
     }
+
+    /* any button that's disabled (we should start using broader overrides) */
+    button[disabled] > svg > path {
+      fill: @overlay0;
+    }
+
+    /* any default button with no styling */
+    button[role*="button"] > svg > path[fill*="hsl(211, 20%, 56%)"] {
+      fill: @overlay2;
+    }
+
+    /* exception for the like button apparently... */
+    button[role*="button"] > div > svg > path[fill*="hsl(211, 20%, 56%)"] {
+      fill: @overlay2;
+    }
+
+    /* sitewide text inputs */
+    /* the divs are styled separately from the input areas */
+    div[style*="border-color: rgb(46, 64, 82)"] {
+      border-color: @surface1 !important;
+    }
+
+    div[style*="border-color: rgb(32, 139, 254)"] {
+      border-color: @accent !important;
+    }
+
+    /* disabled button with accent color */
+    button[style*="background-color: rgb(14, 73, 137)"] {
+      background-color: fade(@accent, 30%) !important;
+    }
+
+    /* image preview borders across app */
+    [style*="border-color: rgb(74, 97, 121)"] {
+      border-color: @surface2 !important;
+    }
+
+    /* explore tab buttons */
+
+    [style*="background-color: rgb(142, 9, 40)"] {
+      background-color: @red !important;
+
+      div {
+        color: @crust !important;
+      }
+
+      svg > path {
+        fill: @crust;
+      }
+    }
+
+    /* TODO: new which is green but I can't get the right class cause nothing on bsky is a new trending topic atm */
+
+    /* subtext across the site */
+    div[style*="color: rgb(147, 165, 183)"] {
+      color: @overlay2 !important;
+    }
+
+    /* random ui lines across the site */
+    div[style*="background-color: rgb(54, 75, 97)"] {
+      background-color: @surface1 !important;
+    }
+
+    /* round icon backgrounds (e.g. feeds) */
+    div[style*="background-color: rgb(21, 52, 85)"] {
+      background-color: @surface0 !important;
+    }
```

Comment:
im *preetty* sure i had a selector for those (or another selector was doing those), but still +1